src/main/scala/org/apache/spark/scheduler/cluster/armada/ArmadaClusterManagerBackend.scala

```diff
@@ -76,7 +77,7 @@ private[spark] class ArmadaClusterSchedulerBackend(
     val host = if (urlArray(1).startsWith("/")) urlArray(1).substring(2) else urlArray(1)
     val port = urlArray(2).toInt
 
-    val driverAddr = sys.env("SPARK_DRIVER_BIND_ADDRESS")
+    val driverAddr = sys.env("ARMADA_SPARK_DRIVER_SERVICE_NAME")
```

Comment:
This is updating the submitJob() method which has been moved to ArmadaClientApp, right?  I think this method is no longer used, so we should remove it.  I'd rather not maintain an unused method.

Reply:
I'll remove it for now, but at some point we want the back-end to be able to add new executors right?

Reply:
That's certainly a valid concern, but I don't think we'll ever want two separate methods that do the same complicated thing.

If we get to that point, I'm pretty sure we'll factor the code into a library so it won't be here either.  Thanks for removing it!